### ISSUE TITLE
tdbloader2 has been renamed tdb1.xloader

### ISSUE THREAD

**Issue by andrefs** [bergos/jena-tdb#4]
This module throws an error when trying to load an RDF file:

```
$ node index.js 
###PROJECT_PATH/node_modules/jena-tdb/Endpoint.js:50
        stream.destroy(new Error(`child process exited with code: ${code} ${errorMessage}`))
                       ^

Error: child process exited with code: 1 tdbloader2 has been renamed tdb1.xloader

    at ChildProcess.<anonymous> (###PROJECT_PATH/node_modules/jena-tdb/Endpoint.js:50:24)
    at ChildProcess.emit (node:events:390:28)
    at maybeClose (node:internal/child_process:1062:16)
    at Socket.<anonymous> (node:internal/child_process:448:11)
    at Socket.emit (node:events:402:35)
    at Pipe.<anonymous> (node:net:687:12) {
  bufferedData: ''
}

Node.js v17.2.0
```

Apparently there was some kind of renaming of TDB utils to distinguish between the two (incompatible) versions: https://jena.apache.org/documentation/tdb2/

Would you consider:

1. Changing the call from `tdbloader2` to `tdb2.tdbloader` [here](https://github.com/bergos/jena-tdb/blob/923fbd530eef0d5a86e9a0b18a7ca8561a7e98d6/Endpoint.js#L66)? I guess this would break legacy code relying on the older version of the TDB loader.

1. Adding something like a `tdbVersion` option (String) to be passed to the client. That string would be called by the `exec()` function.

Otherwise, I can fork your module to `jena-tdb2`  or similar if you prefer.

Thanks!

**Comment by andrefs** [comment]
These lines would also need a similar solution:
https://github.com/bergos/jena-tdb/blob/923fbd530eef0d5a86e9a0b18a7ca8561a7e98d6/StreamQuery.js#L17
https://github.com/bergos/jena-tdb/blob/923fbd530eef0d5a86e9a0b18a7ca8561a7e98d6/StreamQuery.js#L42
https://github.com/bergos/jena-tdb/blob/923fbd530eef0d5a86e9a0b18a7ca8561a7e98d6/StreamStore.js#L25

**Comment by bergos** [comment]
@andrefs I don't think this package needs to support outdated Jena versions. Let's make a breaking change and release it only with up-to-date filenames as 0.2.0. If someone wants to use it with an older version of Jena, it's still possible to use version 0.1.0 of this package. A PR would be very welcome.

**Comment by andrefs** [comment]
Great, I fully agree!

I already have most changes implemented on a fork. 
I'll double check to see if there's anything missing and then I'll open a PR :+1:
